test/helper.py

```diff
@@ -11,7 +12,9 @@ def check_evaluation(str_expr: str, str_expected: str, message=""):
     result = session.evaluate(str_expr)
     expected = session.evaluate(str_expected)
 
+    print(time.asctime())
```

Comment:
Are these for debugging?

Reply:
Yes, it is for seeing where the time is getting spent. There are probably more elaborate ways. If you want to do something more sophisticated with elapsed time, please go ahead.

This test is starting to take a long time and will grow a lot more. 

Since py.test doesn't show this output unless you use `-s` my thought rigtht now is to leave all of this in since wanting to know where the time is spent will be an on-going question. 

Also in the back of my mind is to have some flag to separate quick from long-running tests and this could be something could be done here with verbose logging. 

Again, if this is stuff you feel strongly about feel free to adjust the code.